### ISSUE TITLE
Native support on Windows

### ISSUE THREAD

**Issue by jmp75** [fastai/nbdev#1120]
The documentation is clear about `nbdev` not being supported natively on Windows, and a few prior issues. I'd like to understand why a bit better. 

I personally work from Linux, but intended to promote nbdev to colleagues, most of them working on Windows. Aside from managed IT restrictions to use WSL, I think for many that would be a step too far in practice.

I had a closer look at the nbdev codebase to see if I could easily second guess why it does not support Windows. I'd appreciate a few dot points of the stumbling blocks. Jeremy may have mentioned it in a recent video as I recall but cannot remember where. 

Happy to help if this is feasible, and to receive guidance on how to help.

**Comment by hamelsmu** [comment]
Can you let us know what issues you are running into specifically on Windows?

**Comment by jph00** [comment]
> @jph00 <https://github.com/jph00> do you happen to remember anything about Windows
> 

There's no reason things can't work on Windows -- it's just not something that anyone has taken the time to get working yet. So PRs to make things work would be most welcome!

**Comment by jmp75** [comment]
@hamelsmu  the crux of what I encountered with my colleague was that the command like `nbdev_export` _seemed_ not available from a DOS command prompt.  **However**, I cannot replicate the same issue as my colleague's. We may have bumped into a "black swan" and I jumped to the conclusion "Oh, that's why they say nbdev does not support Windows".

## My colleague's Windows OS:

My colleague added to his conda environment the `nbdev` 2.2.10 package from  conda-forge

From the DOS terminal, with the conda environment activated, `where nbdev_export` failed to find the command.  That really surprised me. After a bit of digging it looked like non-executable shell scripts were under the `Scripts` folder in the conda environment. I'll open a separate issue if we manage to reproduce that. 

## Another Windows OS:

I follow apparently the very same install procedure, and  `where nbdev_export` does return `C:\Users\xxxyyy\Miniconda3\envs\bm\Scripts\nbdev_export.exe`, a binary of some sort, and this launches the python script `nbdev_export.py` as expected.

## Next step

I'll get myself set up on a Windows machine to develop with `nbdev` and see if I can bump into what's "really" not working yet, and if I can contribute. 

In parallel to that, I'll approach a couple of potential users and document impedance to  adoption.
